flask_github.py

```diff
@@ -24,6 +24,45 @@
 null_handler = logging.NullHandler()
 _logger.addHandler(null_handler)
 
+#: MIME type of JSON
+JSON_MIMETYPE = 'application/json'
+
+#: Generic (default) MIME type if response headers are not set
+GENERIC_MIMETYPE = 'application/octet-stream'
+
+
+def is_valid_response(response):
+    """Returns ``True`` if response ``status_code`` is not an error type,
+    raises :class:`GitHubError` otherwise.
+
+    :param response: :class:~`requests.Response` object to check
+    :type response: :class:~`requests.Response`
+    :returns: ``True`` if response ``status_code`` is not an error type,
+              ``False`` otherwise.
+    :rtype bool:
+    """
+    if response.status_code < 400:
+        return True
+    raise GitHubError(response)
```

Comment:
If a function's name starts with ```is_``` it should return a ```bool```. You are using this function below as ```assert is_valid_response(response)```. This creates an ambiguity. Either:
* Remove assert and rename the function to ```check_valid_response ```
* Return ```False``` instead of raising ```GitHubError` and raise outside where it is called.